src/main/java/com/google/api/codegen/viewmodel/DynamicLangXApiView.java

```diff
@@ -18,6 +18,7 @@
 import com.google.auto.value.AutoValue;
 import java.util.List;
 import javax.annotation.Nullable;
+import javax.validation.constraints.Null;
```

Comment:
This appears to be unused

Reply:
oops forgot to format! Should be fixed now.